eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
+automountServiceAccountToken: true
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default
```

Comment:
I think you want a better name for this. Perhaps clusterloader2-cluster-admin (`<serviceaccountname>-<rolename>` or even just clusterloader2.

Reply:
sgtm